### DISCUSSION TITLE
Authentication Errors in "Get Entity," "Search Entity," "Update Entity," and "Revoke Entity" Operations

### DISCUSSION THREAD

**Discussion by shivamg9** [Sunbird-RC/community#868] [Issues]
This is my auth_token: 
eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJIY3VmbHNDZHI0dG1tWFRZcHRTRC1IX0p2R0poWm44VGtCTy1xZW1uUU13In0.eyJleHAiOjE3MDA2NDcwODksImlhdCI6MTcwMDY0Njc4OSwianRpIjoiZTQ5ZjBkODUtNTUzYy00YzNiLWFhMTQtZTUyYjIwYzNjOTlmIiwiaXNzIjoiaHR0cDovL2tleWNsb2FrOjgwODAvYXV0aC9yZWFsbXMvc3VuYmlyZC1yYyIsImF1ZCI6ImFjY291bnQiLCJzdWIiOiIzYTM1NzM1Yy1lOWM1LTRkMzMtYWQzYy0wMzM4OTUwN2ZjODkiLCJ0eXAiOiJCZWFyZXIiLCJhenAiOiJhZG1pbi1hcGkiLCJzZXNzaW9uX3N0YXRlIjoiMTBiMWY1NDYtYTdhNy00ODU0LWFmMzMtZDY5YWE2NTVhZDcyIiwiYWNyIjoiMSIsInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJvZmZsaW5lX2FjY2VzcyIsImRlZmF1bHQtcm9sZXMtc3VuYmlyZC1yYyIsInVtYV9hdXRob3JpemF0aW9uIiwiU3R1ZGVudCJdfSwicmVzb3VyY2VfYWNjZXNzIjp7ImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyIsInZpZXctcHJvZmlsZSJdfX0sInNjb3BlIjoiZW1haWwgcHJvZmlsZSIsImVtYWlsX3ZlcmlmaWVkIjpmYWxzZSwicHJlZmVycmVkX3VzZXJuYW1lIjoic2hpdmFtIiwidXNlcm5hbWUiOiJzaGl2YW0ifQ.OEf5iYsQKNmmwGyLtTEkM3qmYIC0DfBlQvnBqgo0dWKCgMlQb3hpCh0haWQnry-py5ytgZQ3P3WKZsedwmIk5dXGjLaNhwH_SeVR7xvRQI21DaeXRl5Q5GyMXhHV_G35TSVfNcE8GNpoFC0DlCJsud23CgEBnMa8ke2QLViWiSWIE4tqk-gJ-m_On1upmCTRTFpX5k47zWChItB3-DwdvVffbLWimO8o2ZZY1m-EuwEPkVLIcA-tyv8yeluX43359d_GkxbdXnEM7100Gqu68FDG5ZQPPhrFCMXmFXPL93YR4WL_GeKQ9sSQx0FtASOpf9UhGsc-EaDdDg11GkzV4w

When attempting to use the "Get Entity" operation, I'm receiving a "404 Not Found" error with the message: "The requested resource could not be found but may be available again in the future. Subsequent requests by the client are permissible." Here is the response:
{
    "id": "sunbird-rc.registry.get",
    "ver": "1.0",
    "ets": 1700646794038,
    "params": {
        "resmsgid": "",
        "msgid": "e818f2f5-3be1-4d1e-988c-4a77e2a23df4",
        "err": "",
        "status": "UNSUCCESSFUL",
        "errmsg": "No record found"
    },
    "responseCode": "OK"
}

And when attempting "Search Entity," "Update Entity," or "Revoke Entity," I'm encountering a "401 Unauthorized" error, which is similar to a 403 Forbidden status. The error message states: "Similar to 403 Forbidden, but specifically for use when authentication is possible but has failed or not yet been provided. The response must include a WWW-Authenticate header field containing a challenge applicable to the requested resource."

**Comment by manukalia29** [comment]
@holashchand Could you please help with this?
